app/Services/PirepService.php

```diff
@@ -148,6 +148,18 @@ public function prefile(User $user, array $attrs): Pirep
             throw new AircraftNotAvailable($pirep->aircraft);
         }
 
+        // See if this aircraft is being used by another user's active simbrief ofp
+        if (setting('simbrief.block_aircraft')) {
```

Comment:
set the default value to `false` in this. Same thing in the `settings.yaml`

Reply:
Done, it was already `false` at `settings.yaml`. Was it wrong or do i need to define it in a different way ?

Reply:
Ah sorry - I must have missed that somehow. Thanks!

Reply:
No problem and my pleasure as you know :)